sim/hunter/runes.go

```diff
@@ -199,4 +200,17 @@ func (hunter *Hunter) applyLockAndLoad() {
 			}
 		},
 	})
+}
+
+func (hunter *Hunter) applyRaptorFury() {
+	if !hunter.HasRune(proto.HunterRune_RuneBracersRaptorFury){
+		return
+	}
+
+	hunter.RaptorFuryAura = hunter.GetOrRegisterAura(core.Aura{
+		Label:     "Raptor Fury Buff",
+		ActionID:  core.ActionID{SpellID: 415358},
```

Comment:
🔨 Tiny knit but you can pass `int32(proto.HunterRune_RuneBracersRaptorFury)` here in place of the ID because they're the same value 😄

Reply:
all in favor of cleaning up my code! I'll look at using your suggestion